ironfish-cli/src/commands/peers/show.ts

```diff
@@ -27,7 +27,8 @@ export class ShowCommand extends IronfishCommand {
   async start(): Promise<void> {
     const { args } = await this.parse(ShowCommand)
 
-    const identity = (args.identity as string).trim()
+    // TODO(mat): Move trim into parse function like the others
+    const identity = args.identity.trim()
```

Comment:
Do we need to trim args at all? Maybe it's shell dependent, but I imagine oclif needs to separate on whitespace to parse args earlier on

Reply:
We could _probably_ get away without it, but it is possible to do something like `peers:show "     0x89a73b38a99c9    "` (note the enclosing quotes). Admittedly kind of a silly edge-case, but an edge-case nonetheless.

What I'm thinking of doing in a follow-up PR is introducing a `TrimmedStringArg` to avoid all the duplicated boiler plate and make it something we don't have to think about